### PR TITLE
[Javascript] Add support for _created field from v2 API

### PR DESCRIPTION
**Pull request by vavido**
The [v2 thing](https://www.eclipse.org/ditto/basic-thing.html#api-version-2) model can contain a `_created` field. 
Up until now, the internal `Thing`  model of the client simply discarded this field, making it impossible to get the creation date.
This PR introduces a fix for this problem

### CODE REVIEW

File: javascript/lib/api/tests/model/things.model.spec.ts [hunk 1]
```diff
@@ -181,6 +183,7 @@ describe('Thing', () => {
     expect(thing.name).toEqual('Testthing');
     expect(thing.acl).toEqual(acl);
     expect(thing.definition).toEqual('example:test:definition');
+    expect(thing.created).toEqual('2017-06-01T00:01:57Z');
```
Comment (ffendt) [inline javascript/lib/api/tests/model/things.model.spec.ts:187]: Should also be `_created`

File: javascript/package.json [hunk 2]
```diff
@@ -21,7 +21,7 @@
     "lint": "lerna run lint --stream"
   },
   "devDependencies": {
-    "lerna": "^3.16.4"
+    "lerna": "^3.22.1"
```
Comment (ffendt) [inline javascript/package.json:24]: Were there any problems with 3.16.4?

File: javascript/lib/api/src/model/things.model.ts [hunk 3]
```diff
@@ -63,8 +63,8 @@ export class Thing extends EntityWithId {
   }
 
   public toObject(): object {
-    const featuresObj = Features.toObject(this.features);
-    const aclObj = Acl.toObject(this._acl);
+    const featuresObj = this.features ? Features.toObject(this.features) : undefined;
```
Comment (ffendt) [inline javascript/lib/api/src/model/things.model.ts:66]: On master, it's
```
const featuresObj = Features.toObject(this.features);
const aclObj = Acl.toObject(this._acl);
```
Do we need the extra check?

**Review by ffendt — CHANGES_REQUESTED** [review]
Hi  @vavido,
thanks for your PR. I added some comments.

File: javascript/package.json (hunk 2 above)
Comment (vavido) [reply javascript/package.json:24]: No, I'm not sure why the version is upgraded tbh.

File: javascript/lib/api/src/model/things.model.ts (hunk 3 above)
Comment (vavido) [reply javascript/lib/api/src/model/things.model.ts:66]: Actually, no, the toObject method should be able to handle undefined

**Review by ffendt — APPROVED** [review]

**Comment by ffendt** [comment]
Thanks for applying the changes, looks good to me 👍

**Pull request merged**